USE-CASES/use-case-opc-ua-binding.md

```diff
@@ -0,0 +1,82 @@
+ 
+
+### Expected Devices:
+
+Typically automation devices or server endpoints that are able to host an OPC UA server (controller, gateways / edges, etc). 
+
+### Dependencies - Affected WoT deliverables and/or work items:
+There are some experiences of OPC-UA bindings in previous WoT PlugFests and there is a sample binding implementation in [node-wot](https://github.com/eclipse/thingweb.node-wot/). However, there needs to be a formal definition to map the interaction affordances of a TD to OPC-UA. In that context an official OPC-UA Binding Note document schould be developed that can be used as official reference to design Thing Descriptions for OPC-UA use cases. 
+
+
+### Description
+A bottling line consists of a filling module (switchable between 2 fillers and 4 fillers), a capping module, a labeling module, and a transport system. The production line is provided via an OPC-UA endpoint for control and monitoring purposes.
+
+![image](https://user-images.githubusercontent.com/13832739/140924270-9ca88444-f48c-4978-b26b-ae5bf83303e8.png)
+
+In the context of enhancing productivity and sustainability, the goal is to operate the bottling line in such a way that production is further increased when sufficient or surplus renewable energy is available. 
+
+The backend system checks periodically a Smart Grid endpoint (via HTTP) how the current power production is and how much renewable energy is produced. 
+
+Based on the bottling line's current power consumption, which is measured via Modbus, the backend system decides to increase productivity when surplus renewable energy is available.
+
+In doing so, the backend system interacts via OPC-UA to release the 4 fillers of the filling module and increases the speed of the transport system.
+
+If the backend system detects that less renewable energy is being produced, it will initiate standard production and reduce the transport speed and return the 2 fillers of the filling module.
+
+
+### Security Considerations:
+
+OPC-UA has different security modes (sign and/or encrypted, policies, and authentication). Those should be addressed and described in Thing Descriptions with a standardized vocabulary definition. 
```

Comment:
additional considerations may apply if a web bridge is created using WoT servients

Reply:
new version covers this points. thanks